comp/snmpscan/impl/devicescan.go

```diff
@@ -22,7 +22,7 @@ func (s snmpScannerImpl) RunDeviceScan(snmpConnection *gosnmp.GoSNMP, deviceName
 		return err
 	}
 
-	deviceID := deviceNamespace + ":" + snmpConnection.LocalAddr
+	deviceID := deviceNamespace + ":" + snmpConnection.Target
```

Comment:
👍

Reply:
@FlorianVeaux updated this in [d4a47da](https://github.com/DataDog/datadog-agent/pull/29549/commits/d4a47da20e8b8a83aef6a2403256648f0186f3ce) to use the device ip address passed in the scan params instead of the gosnmp target